src/components/Dashboard/dash-summary-styles.postcss

```diff
@@ -29,6 +29,14 @@
        margin-bottom: 0;
      }
     }
+
+    .text-legend-title {
```

Comment:
I wanted to customize only the size for the dashboard mobile resolution, but if I changed it, it would change in cascade in the rest of the resolutions, that's why it seems a new text class but the rest is inherited from the already existing.